pkg/api/handlers/libpod/manifests.go

```diff
@@ -3,61 +3,122 @@ package libpod
 import (
 	"context"
 	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"net/http"
+	"net/url"
+	"strings"
 
 	"github.com/containers/image/v5/docker/reference"
 	"github.com/containers/image/v5/manifest"
 	"github.com/containers/image/v5/types"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/schema"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
 	"github.com/containers/podman/v3/libpod"
 	"github.com/containers/podman/v3/pkg/api/handlers"
 	"github.com/containers/podman/v3/pkg/api/handlers/utils"
 	api "github.com/containers/podman/v3/pkg/api/types"
 	"github.com/containers/podman/v3/pkg/auth"
 	"github.com/containers/podman/v3/pkg/domain/entities"
 	"github.com/containers/podman/v3/pkg/domain/infra/abi"
-	"github.com/gorilla/schema"
-	"github.com/opencontainers/go-digest"
-	"github.com/pkg/errors"
+	"github.com/containers/podman/v3/pkg/errorhandling"
 )
 
 func ManifestCreate(w http.ResponseWriter, r *http.Request) {
 	runtime := r.Context().Value(api.RuntimeKey).(*libpod.Runtime)
 	decoder := r.Context().Value(api.DecoderKey).(*schema.Decoder)
 	query := struct {
-		Name  []string `schema:"name"`
-		Image []string `schema:"image"`
-		All   bool     `schema:"all"`
+		Name   string   `schema:"name"`
+		Images []string `schema:"images"`
+		All    bool     `schema:"all"`
 	}{
 		// Add defaults here once needed.
 	}
+
+	// Support 3.x API calls, alias image to images
+	if image, ok := r.URL.Query()["image"]; ok {
```

Comment:
Above the schema listed the value as "images" here you have it as image?  Are they different?

Reply:
image and images are aliases.